tests/backup/backup_share_test.go

```diff
@@ -4215,3 +4215,400 @@ var _ = Describe("{SwapShareBackup}", func() {
+				userBackupLocationUID := uuid.New()
+				err = CreateBackupLocationWithContext(provider, userBackupLocationName, userBackupLocationUID, userCloudCredentialName, userCloudCredentialUID, getGlobalBucketName(provider), orgID, "", "", nonAdminCtx)
+				log.FailOnError(err, "failed to create backup location %s using provider %s for the user", userBackupLocationName, provider)
+				userBackupLocationMap[userBackupLocationUID] = userBackupLocationName
+			}
+		})
+		Step(fmt.Sprintf("Create source and destination cluster from the user %s", infraAdminUser), func() {
+			log.InfoD(fmt.Sprintf("Creating source and destination cluster from the user %s", infraAdminUser))
+			nonAdminCtx, err := backup.GetNonAdminCtx(infraAdminUser, commonPassword)
+			log.FailOnError(err, "failed to fetch user %s ctx", infraAdminUser)
+			err = CreateApplicationClusters(orgID, "", "", nonAdminCtx)
+			log.FailOnError(err, "failed create source and destination cluster from the user %s", infraAdminUser)
+			clusterStatus, err := Inst().Backup.GetClusterStatus(orgID, SourceClusterName, nonAdminCtx)
+			log.FailOnError(err, fmt.Sprintf("Fetching [%s] cluster status", SourceClusterName))
+			dash.VerifyFatal(clusterStatus, api.ClusterInfo_StatusInfo_Online, fmt.Sprintf("Verifying if [%s] cluster is online", SourceClusterName))
+			userSourceClusterUID, err = Inst().Backup.GetClusterUID(nonAdminCtx, orgID, SourceClusterName)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Fetching [%s] cluster uid", SourceClusterName))
+		})
+		Step(fmt.Sprintf("Take in progress backup of applications from the user %s", infraAdminUser), func() {
+			log.InfoD(fmt.Sprintf("Taking in progress backup of applications from the user %s", infraAdminUser))
+			nonAdminCtx, err := backup.GetNonAdminCtx(infraAdminUser, commonPassword)
+			log.FailOnError(err, "failed to fetch user %s ctx", infraAdminUser)
+			var wg sync.WaitGroup
+			createBackup := func(backupName string, namespace string) {
+				defer GinkgoRecover()
+				defer wg.Done()
+				for backupLocationUID, backupLocationName := range userBackupLocationMap {
+					_, err = CreateBackupByNamespacesWithoutCheck(backupName, SourceClusterName, backupLocationName, backupLocationUID,
+						[]string{namespace}, map[string]string{}, orgID, userSourceClusterUID, "", "", "", "", nonAdminCtx)
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of backup %s of namespace %s", backupName, namespace))
+					break
+				}
+			}
+			for _, namespace := range appNamespaces {
+				for i := 0; i < numberOfBackups; i++ {
+					backupName := fmt.Sprintf("%s-%s-%d-%v", BackupNamePrefix, namespace, i, time.Now().Unix())
+					wg.Add(1)
+					go createBackup(backupName, namespace)
+					userInProgressBackupNames = append(userInProgressBackupNames, backupName)
+				}
+			}
+			wg.Wait()
+			log.Infof("The list of in progress user backups taken are: %v", userInProgressBackupNames)
+		})
```

Comment:
I might be missing a step to verify backup by non-admin user is listed from admin pages while backup is in progress state.

Reply:
@kphalgun-px , As we discussed before it's an important step for orphaned backup delete. Ive added a comment where you need to add it.

Reply:
Done.